### ISSUE TITLE
Disable Expand to Current Folder in Navigation Pane

### ISSUE THREAD

**Issue by thewaliii** [oliverschwendener/ueli#379]
How can I stop the expansion of the current folder in the navigation pane? 
When I use "Simple file search" and I navigate to a folder, this happens.

![2020-05-09_14-18-48](https://user-images.githubusercontent.com/48418015/81473672-5449cf00-9200-11ea-94a4-0ffd219ceea6.png)

**Comment by oliverschwendener** [comment]
Is this a duplicate of #350 ?

**Comment by Waleedov** [comment]
I checked that, but that is about (Navigation pane) as a whole? I don't have that issue.
My case is specific about expanding (unfolding) the current folder.

**Comment by AmrBinBashir** [comment]
It's the same issue as Oliver stated, What causes the navigation pane to show up , is the same issue that causes expand to current folder. I tried to look what causes this issue a while back and I think I found the issue and it is electron shell api. The underlying c++ method they use to open files and folders causes the navigation pane to expand. The only way I can think of to solve this issue is by going back to use powershell to open folders but that would be much slower.
